ailment/expression.py

```diff
@@ -29,7 +29,7 @@ def has_atom(self, atom, identity=True):
             return self.likes(atom)
 
     def __eq__(self, other):
-        return self.likes(other) and self.idx == other.idx
+        return isinstance(other, Expression) and self.idx == other.idx
```

Comment:
Why?

Reply:
@ltfish I need to find the example again. This caused an infinite loop when doing `likes` on a call

Reply:
@ltfish why would it make sense to have a `likes` inside an `__eq__`, does this not destroy the point of `likes`?

Reply:
Alright, @ltfish, nvm. I found the loop. I'm just an idiot.